### PR TITLE
Create get_admins.py

### PR DESCRIPTION
**Pull request by muhammedfurkan**

### CODE REVIEW

File: pyrobot/plugins/get_admins.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import pyrogram
+from pyrogram import Filters, Client
+from pyrobot import app, cmd
```
Comment (SpEcHiDe) [inline pyrobot/plugins/get_admins.py]: this will fail, right?

Comment (muhammedfurkan) [reply pyrobot/plugins/get_admins.py]: no it working..

Comment (SpEcHiDe) [reply pyrobot/plugins/get_admins.py]: there is no "cmd" ?? How will it work? 🙄🙄

Comment (muhammedfurkan) [reply pyrobot/plugins/get_admins.py]: https://github.com/SpEcHiDe/PyroGramUserBot/pull/5

 fixed

Comment (SpEcHiDe) [reply pyrobot/plugins/get_admins.py]: why you opening multiple PRs? :thinking:

Comment (muhammedfurkan) [reply pyrobot/plugins/get_admins.py]: how can I only one pr ?

**Pull request closed without merging**